kratos/utilities/read_materials_utility.cpp

```diff
@@ -64,6 +71,14 @@ ReadMaterialsUtility::ReadMaterialsUtility(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void ReadMaterialsUtility::ReadMaterials(Parameters MaterialData)
```

Comment:
wouldn't it be better to remove overall the constructor 

      ReadMaterialsUtility( Params, model )

and to have two methods 

      ReadMaterials(std::string filename )
 
and 

      ReadMaterials(Parameters MaterialData)

Reply:
Then we will remove the current method breaking the retrocompatibility